src/action.ml

```diff
@@ -374,8 +374,8 @@ module Unexpanded = struct
         | Left x -> map x
         | Right template ->
            match To_VE.expand dir template ~f with
-           | Left  e -> special dir e
-           | Right s -> generic dir s
+           | SW.Expand.Full.Expansion  e -> special dir e
```

Comment:
Do we need the path here? The return type of `To_VE.expand` should be enough for the type-based disambiguation

Reply:
you're right, we don't need any of them.